sb3_contrib/ppo_recurrent/ppo_recurrent.py

```diff
@@ -378,6 +378,9 @@ def train(self) -> None:
                     # Convert discrete action from float to long
                     actions = rollout_data.actions.long().flatten()
 
+                # Convert mask from float to boole
```

Comment:
```suggestion
                # Convert mask from float to bool
```

(use either Bool or Boolean)